### ISSUE TITLE
Package request: tinc

### ISSUE THREAD

**Issue by hetykai** [termux/unstable-packages#42]
Can [tinc](https://www.tinc-vpn.org/) be implemented in termux?

**Comment by xeffyr** [comment]
Package is now available.

You can obtain it with these commands:
```
pkg install unstable-repo
pkg install tinc
```

**Comment by shynome** [comment]
@xeffyr hi, can you tell me where to find the relevant documentation ? 
dose it need root ? I can not run it , there are error :
```log
tincd 1.0.35 starting, debug level 5
Could not open /dev/net/tun: No such file or directory
Terminating
```

**Comment by xeffyr** [comment]
> dose it need root ?

Of course.

**Comment by xeffyr** [comment]
> Could not open /dev/net/tun:

`/dev/net/tun` is incorrect tun device path. Should be fixed in 1.0.35-4.
